docs/index.md

```diff
@@ -24,7 +24,8 @@
 </center>
 <!-- markdownlint-restore -->
 
-Starlite is a light, opinionated and flexible ASGI API framework built on top of **[pydantic](https://github.com/samuelcolvin/pydantic)**.
+Starlite is a light, opinionated and flexible ASGI API framework built on top
```

Comment:
I'm not sure we're builtin on top of. I think we can drop this and simply add we support pydantic 100%. The internals are internal.